### ISSUE TITLE
Custom scroll doesn’t work with trackpad on macOS

### ISSUE THREAD

**Issue by vitalytalyzin** [malte-wessel/react-custom-scrollbars#349]
Hello. 
When using the scroll on the trackpad in Chrome on macOS Catalina 10.15.4, default scroll bars appear, the custom scrollbar doesn't work and tracks have `display: none` style. But when you turn off the trackpad or set the global `-webkit-scrollbar` styles, like `body::-webkit-scrollbar { width: 10px, height: 10px;}`, everything works fine. Have anybody face this kind of problem and how did solve it? Is it possible?

**Comment by jianfenkezhan** [comment]
same issue

**Comment by SolidZORO** [comment]
same issue.

**Comment by adamatronix** [comment]
Yes me too. Vertical track gets a display: none, and thumb doesn't even get a calculated height.

**Comment by resthedev** [comment]
Same issue here with trackpad

**Comment by sajera** [comment]
same issue

**Comment by KosolapovR** [comment]
same issue
